packages/genesys-spark-tokens/data/semantic/global.json

```diff
@@ -254,6 +254,32 @@
       "description": "We use this typography style for header's eyebrows, meaning a little text that go just above the header that can give more context on the header below.Eyebrows a re meant to be concise and not full sentences, if you need to give more context try using an subheading.\n\nWe usually pair it with low emphasis colors."
     }
   },
+  "rte": {
```

Comment:
@dohajnal Should RTE specific tokens be found in the global file? Would it be better if they lived with the other RTE tokens?

Reply:
hm yes, this should not be there, I will clean it up with the other change and let you know!

Reply:
These were basically duplicate tokens, so I just removed them from the global.